src/utils/enforcements.ts

```diff
@@ -17,6 +19,33 @@ export function checkEnforcementJSON(test: string): boolean {
   return /^[\],:{}\s]*$/.test(test)
 }
 
+// Returns whether an application should be rate limited
+export async function shouldRateLimit(
+  appID: string,
+  cache: Cache
+): Promise<{ shouldLimit: boolean; numRateLimitedApps: number }> {
+  let obj = {
+    shouldLimit: false,
+    numRateLimitedApps: 0,
+  }
+
+  if (appID.length === 0) {
+    return obj
+  }
+
+  const url = process.env.RATE_LIMITER_URL ?? ''
```

Comment:
the portal-api usually reads the url as a global variable in top of the file or is passed as a function, I personally prefer the later as is more clear where the value is coming from but I recommend one of the two for consistency

Reply:
Thank you for the review. Fixed.